### PR TITLE
Update nrf pacs

### PR DESCRIPTION
**Pull request by diondokter**

### CODE REVIEW

**Review by Dirbaio — APPROVED** [review]
THanks!

bors r+

**Comment by diondokter** [comment]
Ok, this change to USB should be alright. I checked that the variant is correct for each

File: embassy-nrf/src/usb.rs [hunk 1]
```diff
@@ -313,7 +313,7 @@ impl<'d, T: Instance, P: UsbSupply> driver::Bus for Bus<'d, T, P> {
                 }
             })
             .await;
-            regs.eventcause.write(|w| w.ready().set_bit()); // Write 1 to clear.
+            regs.eventcause.write(|w| w.ready().clear_bit_by_one());
```
Comment (Dirbaio) [inline embassy-nrf/src/usb.rs:316]: `clear_bit_by_one`? wtf :o

Comment (diondokter) [reply embassy-nrf/src/usb.rs:316]: Yeah, I saw that as well. It's pretty verbose which is nice :)

Comment (Dirbaio) [reply embassy-nrf/src/usb.rs:316]: IMO the name is not obvious at all what it does. I'd say `write_one_to_clear`, or `clear_by_writing_one` or something. But yeah it's just bikeshedding :shrug:

**Pull request merged**